helmfile/50-applications.yaml

```diff
@@ -404,6 +415,21 @@ releases:
 
 # Workload cluster releases
 {{ if eq .Environment.Name "workload_cluster" }}
+
+# prometheus wc-servicemonitor
+- name: wc-servicemonitor
+  namespace: monitoring
+  labels:
+    app: prometheus-servicemonitor
+    prometheus: sc-prometheus
+  chart: ./charts/prometheus-servicemonitor
+  version: 0.1.1
+  missingFileHandler: Error
+  values:
+  # Note: we want to have alerts for WC since this is wc-reader, even if it is
+  # running in SC.
+  - values/sc-servicemonitor.yaml.gotmpl
```

Comment:
```suggestion
  - values/wc-servicemonitor.yaml.gotmpl
```

Reply:
Also, add this file. I don't want the two prometheuses to share the same values file.